NEWS.md

```diff
@@ -1,11 +1,38 @@
 MXNet Change Log
 ================
+## 0.12.0
+### New Features - Sparse Tensor Support
+  - Added comprehensive support for sparse matrices. See help on `mx.sym.sparse` and `mx.nd.sparse` for more info.
```

Comment:
- Added limited cpu support for two sparse formats for `Symbol` and `NDArray` - `CSRNDArray` and `RowSparseNDArray`
- Added a sparse dot product operator and many element-wise sparse operators
- Added a data iterator for sparse data input - `LibSVMIter`
- Added three optimizers for sparse gradient updates: `Ftrl`, `SGD` and `Adam`
- Added `push` and `row_sparse_pull` with `RowSparseNDArray` in distributed kvstore

Reply:
Should I replace the existing point or add subpoints?

Reply:
Yeah please replace the existing point. `comprehensive support ` is misleading since we dont have gpu support. Thanks!

Reply:
done

Reply:
Ah, all the formats are gone.. :(